scripts/maven/DependencyTree.kt

```diff
@@ -0,0 +1,8 @@
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/** Helper Data class to parse the list of dependencies from maven_intall.json. */
```

Comment:
This is actually a structure definition class, not a helper class (e.g. it has no logic to 'help' with anything, but it can be used to parse the list of dependencies from maven_install).

Reply:
Fixed in the fake package's class.